src/open_producten/conf/base.py

```diff
@@ -47,6 +49,8 @@
     "django.middleware.locale.LocaleMiddleware",
 )
 
+MIDDLEWARE.append("reversion.middleware.RevisionMiddleware")
```

Comment:
Does this also affect API requests? I see in the docs that this will wrap every request

https://django-reversion.readthedocs.io/en/latest/middleware.html

Reply:
Yes, it is needed to create versions when an object is changed through the API.

Reply:
The issue mentions that it should be admin only, but I guess this has been changed then?

Reply:
the version beheer/management should be admin only but versions themself must always be created.

Reply:
check, thanks for explaining